### PR TITLE
Add StarterProjectHealthMetricsModel

### PR DESCRIPTION
**Pull request by lishengbao**

### CODE REVIEW

**Comment by eyehwan** [comment]
https://github.com/oss-compass/community/issues/10

**Review by eyehwan — COMMENTED** [review]
with some minor comments.

File: run.py [hunk 1]
```diff
@@ -52,6 +54,13 @@
    model_organizations = OrganizationsActivityMetricsModel(**kwargs)
    model_organizations.metrics_model_metrics(elastic_url)
 
+   kwargs = {}
```
[the remark this answers is not in the document]
Comment (lishengbao) [reply run.py]: done.

File: compass_metrics_model/metrics_model_lab.py [hunk 2]
```diff
@@ -1 +1,205 @@
...
+                            MAX_BULK_UPDATE_SIZE,
+                            check_times_has_overlap,
+                            create_release_index,
+                            get_medium)
+from .utils import (get_uuid, get_date_list)
+from .utils_lab import (starter_project_health, starter_project_health_decay)
 
+logger = logging.getLogger(__name__)
+
+def is_bot_by_author_name(*author_name):
```
Comment (eyehwan) [inline compass_metrics_model/metrics_model_lab.py:17]: 1. This part should not hard code here, prefer adding an extra config file to record all bot account. 
2. Except for full name of bot, also need to consider pattern, like **bot, to filter out bot.

**Review by eyehwan — CHANGES_REQUESTED** [review]

**Comment by eyehwan** [comment]
I will merge first.

**Pull request merged**